source/boards/work-with-cards.rst

```diff
@@ -4,38 +4,109 @@ Work with cards
-* **Created time/Created by/Last updated time/Last updated by** are good ways order tasks in order of recency.
+Boards supports a wide range of fully customizable property types:
+
+- **Text** can be used to add short notes to a card. An advantage of the text property over card descriptions is that it can be `shown on the board <https://docs.mattermost.com/boards/work-with-cards.html#toggle-properties-shown-on-a-board>`_ without needing to open the card.
+- **Numbers** are useful to capture metrics such as task sizing or effort estimates. Use in conjunction with Calculations to get the most out of the number property type.  
+- **Email** and **Phone** can be used to record contact information.
+- **URL** can be used to provide a link to a pull request or relevant website. Clicking on the box of a URL property will automatically open the link in a new tab on your browser. Hover over the box to surface options to copy or edit the URL.
+- **Select** and **Multi-select** allows you to create a predefined list of options that can be color-coded and displayed as badges on the card to indicate things like status and priority.
+- **Dates** are useful to set and track due dates or milestones. Use the date property to make a card appear on the `Calendar view <https://docs.mattermost.com/boards/work-with-views.html#calendar-view>`_. Set a single date or toggle on the **End date** to set a date range.
+- **Person** provides a quick way to capture user assignments. Note that this is not available in Personal Desktop.
+- **Checkbox** is a toggle property that can be used for assigning simple binary options on a card such as True/False or Yes/No.
+- **Created time/Created by/Last updated time/Last updated by** are predefined system properties to help you audit changes on a card. The names of these properties are customizable, but the values are not.
+
+Rename a property
+-----------------
+
+The default name for a new property is the name of the property type (e.g. **Date**, **URL**).
+To rename a property field, open up a card and select the property name to open an editable field. Enter the new name in the field provided. The change is saved immediately and applied across all cards on the current board.
+
+Change a property type
+----------------------
+
+To change a property type, select the property, then open the **Type** menu and choose a new property type. You’ll be asked to confirm the change from every card on the current board. Changing the type for an existing property will affect values across all cards on the board and may result in data loss.
+
+Delete a property
+-----------------
+
+To delete properties you no longer need, select the property, then choose **Delete**. You’ll be asked to confirm that you want to remove that property from every card on the current board.
+
+.. note:: 
+  
+  Properties are displayed in the order they were created and cannot be re-ordered at this time.
+
+Define a **Select** or **Multi-select** property
```

Comment:
```suggestion
Define a "Select" or "Multi-select" property
```

Reply:
This header with bold text is showing up weird on the live preview. Maybe quotes will render better.